draft-bradley-dnssd-private-discovery.md

```diff
@@ -97,16 +103,22 @@ A probe is sent via multicast to discover friends on the network. A probe contai
 Probe Fields:
 
 * EPK1 (Ephemeral Public Key 1).
-* TS1 (Timestamp 1). See Timestamps (#timestamps).
-* SIG1 (Signature of "Probe" || EPK1 || TS1 || "End").
+* TS1 (Timestamp 1). See Timestamps {{timestamps}}.
+* SIG1 (Signature of "Probe" \|\| EPK1 \|\| TS1 \|\| "End").
 
 When a peer receives a probe, it verifies TS1. If TS1 is outside the time window then it SHOULD be ignored. It then attempts to verify SIG1 with the public key of each of its friends. If verification fails for all public keys then it ignores the probe. If a verification succeeds for a public key then it knows which friend sent the probe. It SHOULD send a response to the friend.
 
+If verification fails for all public keys it MAY send a fake response (see {{response}}) so as to hide information about the friend list. Always sending fake responses allows an adversary to learn the size of one's friend list. Never sending fake responses allows an eavesdropper to learn whether two peers are friends. 
+
+If a verification succeeds for a public key then it knows which friend sent the probe. It SHOULD send a response to the friend, and this response SHOULD be sent after some delay proportional to the size of the friend list. (This is done so as to not leak information about the friend list.)
+
+Probes SHOULD NOT be sent at regular intervals. Doing so may allow passive observes to track devices based on the observed probed cadence.
+
 Message format:
 
 ~~~~
 +0   +-----+---------+
-     |Flags| Type=1  | 1 byte
+     |0|0|0| Type=1  | 1 byte
```

Comment:
This looks weird to me to have 3 zeros. I'm not sure it's obvious that it's a flags field. Maybe it should be F=0?